spec.html

```diff
@@ -245,62 +252,62 @@ <h3>Non-terminals</h3>
                   <tr class="section">
                     <td class="fix nt">subtype</td>
                     <td class="fix op">::=</td>
-                    <td class="fix ex">"\x00"</td>
+                    <td class="fix ex">null_byte</td>
```

Comment:
You should probably used `signed_byte(0)` here rather than `null_byte`. They're not really different, but I think it's more clear because elsewhere we only represent it as `null_byte` in the case of a null terminator. Whereas here, the 0000 0000 bit pattern is not a null terminator, it's just a (somewhat arbitrarily chosen) type code.